app/src/pages/Signup.tsx

```diff
@@ -36,8 +36,6 @@ const AnimatedTitle = styled(Heading).attrs({ as: "h1" })`
 `;
 
 function Signup() {
-  useAuthCheck("/signup");
```

Comment:
ya laaaahwiiii

Reply:
dude think about it , will the user be authenticated if they are trying to signup

Reply:
it's just for PROTECTING routes rg3eha tani lazem😂